src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -3,20 +3,42 @@
 import java.util.Optional;
 import mate.academy.hibernate.relations.dao.ActorDao;
 import mate.academy.hibernate.relations.model.Actor;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
 
 public class ActorDaoImpl extends AbstractDao implements ActorDao {
+    private static final String CAN_NOT_ADD_ACTOR_EXCEPTION = "Can't add actor into DB";
+    private static final String CAN_NOT_GET_ACTOR_FROM_DB_EXCEPTION = "Can't get actor from DB";
+
     public ActorDaoImpl(SessionFactory sessionFactory) {
         super(sessionFactory);
     }
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        try (Session session = factory.openSession()) {
```

Comment:
In this case it's better to use try-catch-finally, to not create redundant code intent by new block. It is also mentioned in common mistakes:
![image](https://github.com/mate-academy/hibernate-relations-hw/assets/82539553/6193543a-241f-4d97-bb5b-092ede8fd12f)